module/importer-ergonode-1/src/Resources/config/services.yml

```diff
@@ -0,0 +1,36 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Ergonode\ImporterErgonode\Application\:
+        resource: '../../Application/*'
+
+    Ergonode\ImporterErgonode\Domain\:
+        resource: '../../Domain/*'
+        exclude: '../../Domain/{Entity}'
+
+    Ergonode\ImporterErgonode\Infrastructure\:
+        resource: '../../Infrastructure/*'
+
+    Ergonode\ImporterErgonode\Infrastructure\Handler\:
+        resource: '../../Infrastructure/Handler/*'
+        tags: ['messenger.message_handler']
+
+    Ergonode\ImporterErgonode\Infrastructure\Resolver\ProductCommandResolver:
+        arguments:
+            - !tagged_iterator 'component.ergonode-importer.product_command_factory_interface'
```

Comment:
Oh, as I see it it is in the configuration. Above compiler passes should be removed.